dev-requirements.txt

```diff
@@ -1,3 +1,4 @@
+-r requirements.txt
```

Comment:
Not sure about that one

Reply:
saves one pip install for devs

Reply:
yeah, but as dev you will do `pip install -e .` and `pip install -r dev-req.txt` so the first one will always install the `requierments.txt` I think it would make more sense to have the setup.py use a proper dev install that does both

Reply:
you are right
my change was propmted by the frustration of failing tests bcs of a new dependency in req.txt that I did not get to work by rerunning pipi install dev-req. Maybe a bit of an overreaction on my part